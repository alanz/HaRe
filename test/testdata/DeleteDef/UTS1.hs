module UTS1 where

type Foo = String

fooLen :: String -> Int
fooLen = length
