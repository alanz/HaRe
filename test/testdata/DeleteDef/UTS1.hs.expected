module UTS1 where

fooLen :: String -> Int
fooLen = length